packages/ergo-lib-wasm/src/blockchain.rs

```diff
@@ -0,0 +1,45 @@
+use crate::declarations::HexString;
+use crate::prelude::*;
+use derive_more::{From, Into};
+use ergo_lib::ergo_chain_types::Header;
+use ergo_wasm_derive::TryFromJsValue;
+use js_sys::JsString;
+use serde::{Deserialize, Serialize};
+use tsify::declare;
+use wasm_bindgen::{prelude::wasm_bindgen, JsCast};
+
+#[declare]
+#[allow(dead_code)]
+type BlockId = HexString;
```

Comment:
Would like your opinion on this @greenhat

Instead of using `wasm_bindgen` for `BlockId` that generates a new class use a type alias.

This produces something like:

```ts
export type HexString = string;

export type BlockId = HexString;

export class NipopowVerifier {
  constructor(genesis_block_id: BlockId);
}

const verifier = new NipopowVerifier("b0244dfc267baca974a4caee06120321562784303a8a688976ae56170e4d175b")
```

So devs don't need to convert to and from a concrete `BlockId` class but we still get some duck typing so devs know that `BlockId` should be a hex string, etc

Reply:
I'm torn on this one. On the one hand, I really like the idea that it prevents passing a box id when tx id is expected. Type alias won't give you that. On the other hand I understand that it creates some friction with conversion. Is there any way to reduce this friction?

Reply:
Yeah, my exact thoughts. If we do use concrete classes is there anything that prevents creating a `TxId` from a box id hex string? Not sure if this would be less likely though 🤷 

Can't really think of any other ways to reduce friction

Reply:
Yes, you're right. In this case it does not prevent, but makes it way more explicit than type alias (you have to construct a TxId from a string). In prevention I meant the case when we have TxId from our API and use in the subsequent API calls.

Reply:
👍 I'll change to explicit